zerver/lib/emoji.py

```diff
@@ -14,6 +14,30 @@
 NAME_TO_CODEPOINT_PATH = os.path.join(settings.STATIC_ROOT, "generated", "emoji", "name_to_codepoint.json")
 CODEPOINT_TO_NAME_PATH = os.path.join(settings.STATIC_ROOT, "generated", "emoji", "codepoint_to_name.json")
 
+# Emoticons and which emoji they should become. Duplicate emoji are allowed.
+# Changes here should be mimicked in `static/js/emoji.js`.
+EMOTICON_CONVERSIONS = {
```

Comment:
@skunkmb Was just going through this PR and I think it would make more sense to export this mapping using `build_emoji` rather than hard coding it here and in `emoji.js`. That way they will not go out of sync. What do you think? Thanks for working on this! :)

Reply:
Also, that's probably a good idea (to have the data structure be sent from server to client via that data structure).  @HarshitOnGitHub can you open an issue for doing that follow-up cleanup?  Not a high priority until we're changing the list, but seems useful.

Reply:
@HarshitOnGitHub just checking in on this; might be a good quick thing for you to fix post release?

Reply:
Sure, I am adding this to my ToDo! :)